src/go/rdctl/cmd/shell.go

```diff
@@ -99,3 +104,20 @@ func setupLimaHome() error {
 	os.Setenv("LIMA_HOME", candidatePath)
 	return nil
 }
+
+func checkLimaIsRunning(commandName string) bool {
+	output, err := exec.Command(commandName, "ls", "0", "--format", "{{.Status}}").CombinedOutput()
+	if err == nil {
+		if strings.HasPrefix(string(output), "Running") {
+			return true
+		} else {
+			fmt.Fprintf(os.Stderr, fmt.Sprintf(
+				`The Rancher Desktop VM needs to be in state "Running" in order to execute 'rdctl shell',
+but it is currently in state "%s". Either run 'rdctl start' or start the Rancher Desktop application first.
+`, strings.TrimRight(string(output), "\n")))
```

Comment:
I don't understand the need for the nested `fmt.Sprintf`, and it is actually problematic if `output` contains any further `%` formating codes.

```suggestion
			fmt.Fprintf(os.Stderr, 
				`The Rancher Desktop VM needs to be in state "Running" in order to execute 'rdctl shell',
but it is currently in state "%s". Either run 'rdctl start' or start the Rancher Desktop application first.
`, strings.TrimRight(string(output), "\n"))
```